### ISSUE TITLE
report train loss and accuracy only?

### ISSUE THREAD

**Issue by chaoshengdong** [dbmptr/EPOSearch#2]
Hi, are the losses and accuracies stored in .pkl for training data only? Shall we report the testing loss and accuracy. Thank you!

**Comment by dbmptr** [comment]
Look at line number 158 in the epo_train.py file. We load from the test_loader, not train_loader. 
The code for recording performance has been adopted from the the Pareto MTL paper by Lin et al.
But, I agree the code should change the naming of the variables.
